starters/micronaut/stove-micronaut-testing-e2e/src/main/kotlin/com/trendyol/stove/testing/BaseApplicationContextInitializer.kt

```diff
@@ -0,0 +1,36 @@
+package com.trendyol.stove.testing
+
+import com.trendyol.stove.testing.e2e.system.annotations.StoveDsl
+import io.micronaut.context.ApplicationContext
+import io.micronaut.context.event.ApplicationEventListener
+import io.micronaut.context.event.StartupEvent
+import io.micronaut.runtime.Micronaut
+
+@StoveDsl
+abstract class BaseApplicationContextInitializer : ApplicationEventListener<StartupEvent> {
```

Comment:
Yo don't need this one I believe.

Reply:
This is a spring-boot initializer approach, and for micronaut it is not applicable.